.github/workflows/ci.yml

```diff
@@ -216,11 +220,10 @@ jobs:
       - name: start abq work
         # abq work _should_ fail because we have failing tests, but we want to ensure it only has a single line of output
         run: |
-          set +e
-          echo "run worker"
-          abq work -n cpu-cores --run-id "$GITHUB_RUN_ID-$GITHUB_RUN_ATTEMPT-${{matrix.ruby}}-${{matrix.gemfile}}" 2>&1 |
-            bin/test-worker-output.rb
+          bin/test-worker-output.rb success
```

Comment:
could run this as a matrix but it's such a pain to update the matrix that I'll punt for now.

If / when we set required status via a script or terraform, we'll expand this to a matrix.